src/sentry/models/eventattachment.py

```diff
@@ -5,7 +5,12 @@
 from django.db import models
 from django.utils import timezone
 
-from sentry.db.models import BoundedBigIntegerField, FlexibleForeignKey, Model, sane_repr
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    FlexibleForeignKey,
+    Model,
+    sane_repr,
+)
```

Comment:
_nit_: Changes in this file seem unrelated could be reverted to minimize the diff.

Reply:
thanks, yeah I used to have attachment type added here until I realized which table I am about to migrate (and until I realized it's on File so we're good for now)